_posts/2024-10-29-dsf-candidates-2025.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Black Python Devs Running for the DSF"
+date: 2024-10-29
+author: Jay Miller
+lang: en
+layout: post
+featured_image: /assets/images/django-foundation-election.webp
+---
+
+The [DSF Board Candidates have been announced](https://www.djangoproject.com/weblog/2024/oct/28/2025-dsf-board-candidates/) and voting is now open for DSF members now until 15 November (Anywhere on Earth).
+
+The DSF Board serve two year terms. "This is an excellent opportunity to help advance Django. They guide the direction of the marketing, governance and outreach activities of the Django community."
```

Comment:
I wanted to make sure that folks learn more about that the board is responsible for. This isn't everything they do but it is a direct quote from the Election announcement.